Bottom: 0259758746426b71cd983a2073616ca4d4a81135
Top:    c395e3da3639d7a6ca503d0910bcba8af2e597d5
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-12-18 22:45:22 +0000

Refresh of hid-common-hidgl-c-initialise-

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 4afaac7..67e610d 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1574,6 +1574,7 @@ GhidDrawMask (int side, BoxType * screen)
   polygon.NoHolesValid = 0;
   if (screen)
     polygon.BoundingBox = *screen;
+  polygon.Flags = NoFlags ();
   SET_FLAG (FULLPOLYFLAG, &polygon);
   common_fill_pcb_polygon (out->fgGC, &polygon, screen);
   poly_Free (&polygon.Clipped);
